src/include/sof/audio/smart_amp/aw_smart_amp.h

```diff
@@ -0,0 +1,128 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2023 Awinic Integrated. All rights reserved.
+ *
+ * Author: Jimmy Zhang <zhangjianming@awinic.com>
+ */
+
+#ifndef __SOF_AUDIO_AW_H__
+#define __SOF_AUDIO_AW_H__
+
+#include <sof/platform.h>
+#include <sof/audio/component.h>
+
+/* Awinic Maximum number of channels for algorithm in */
+#define SMART_AMP_FF_MAX_CH_NUM		2
+/* Awinic Maximum number of channels for algorithm out */
+#define SMART_AMP_FF_OUT_MAX_CH_NUM	4
+/* Awinic Maximum number of channels for feedback  */
+#define SMART_AMP_FB_MAX_CH_NUM		4
+
+#define SMART_AMP_FRM_SZ	48 /* samples per 1ms */
```

Comment:
48 is samples per ms, not frame size

Reply:
Ok. Is there anything we need to change?

Reply:
Should probably rename your define so it makes sense. Also that config is only true in 48k which the system may not always use. Recommend you use some of the audio_stream helpers for this math

Reply:
Ok, we'll modify the algorithms once they're stable